### PR TITLE
Add MKPC presence

### PR DESCRIPTION
**Pull request by trains2050**
Ive added a presence for MKPC. Proof:
image

### CODE REVIEW

File: websites/M/MKPC/dist/metadata.json [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+      "fr": "Vous connaissez certainement Mario Kart, le jeu de course le plus fun de tous les temps ! Mario Kart PC reprend les mêmes principes que le jeu original mais il est jouable sur navigateur, et gratuitement.  La plupart des modes issus de Mario Kart ont été repris : Grand Prix, courses VS, batailles de ballons, contre-la-montre... Et un dernier mode inédit : l'éditeur de circuits ! Placez les lignes droites et les virages, ajoutez les objets, insérez des accélérateurs... Tout est personnalisable ! Votre imagination est la seule limite ! Vous pouvez également partager vos créations et essayer celles des autres grâce à l'outil de partage. Plusieurs milliers de circuits ont déjà été partagés !  Enfin, il est possible d'affronter les joueurs du monde entier grâce au mode multijoueurs en ligne ! Grimpez dans le classement et devenez champion du monde !"
+    },
+    "service": "MKPC",
+    "version": "1.0.0",
+    "logo": "https://imgur.com/T2WTLEM.jpeg",
```
Comment (Bas950) [inline websites/M/MKPC/dist/metadata.json:20]: logo must be 512x512

**Review by Bas950 — CHANGES_REQUESTED** [review]

File: websites/M/MKPC/dist/metadata.json (hunk 1 above)
Comment (trains2050) [reply websites/M/MKPC/dist/metadata.json:20]: done

Comment (trains2050) [reply websites/M/MKPC/dist/metadata.json:20]: its the same link, just updated via imgur editor

**Review by Bas950 — APPROVED** [review]

**Review by ririxidev — APPROVED** [review]

**Comment by ririxidev** [comment]
ffs, just noticed that you added contributor after clicking approved button, don't do it next time bruh

**Pull request merged**